data/src/main/java/me/androidbox/data/worker_manager/util/CreatePhotoMultipart.kt

```diff
@@ -0,0 +1,7 @@
+package me.androidbox.data.worker_manager.util
+
+import okhttp3.MultipartBody
+
+interface CreatePhotoMultipart {
```

Comment:
Do you really need this interface?

Reply:
Also it rather sounds like a function and not an interface